scripts/engine.js

```diff
@@ -18,8 +18,23 @@ function loadEvenOdd() {
   document.getElementById("parkExplanation").innerHTML = parkingDeets.get('parkExplanation');
 }
 
+function loadLights() {
+  const now = new Date();
+  if (now.getMonth() == 11 && now.getDate() <= 25) {
+    const lines = document.querySelectorAll(".line")
+    for (var i = 0; i < lines.length; i++) {
+      lines[i].style.display = "none"
+    }
```

Comment:
With these two loops, I think I'd prefer something more succinct using some of the iteration features of JavaScript:

```javascript
document.querySelectorAll('.line').forEach(function(el) {
  el.style.display = 'none';
});
```

Reply:
(Incidentally, after bringing this in there's a chance I'll reduce it to a single strand of lights to make sure  important information doesn't get pushed down the page. So some of the looping and such will probably evolve to something else. [Or maybe not. The lights scale pretty nicely on the phone!])